client/my-sites/site-settings/manage-connection/site-ownership.jsx

```diff
@@ -0,0 +1,121 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLegend from 'components/forms/form-legend';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import SectionHeader from 'components/section-header';
+import Site from 'blocks/site';
+import QueryJetpackConnection from 'components/data/query-jetpack-connection';
+import QueryJetpackUserConnection from 'components/data/query-jetpack-user-connection';
+import { getSelectedSite, getSelectedSiteId } from 'state/ui/selectors';
+import { isJetpackSite } from 'state/sites/selectors';
+import {
+	isJetpackSiteConnected,
+	isJetpackSiteInDevelopmentMode,
+	isJetpackUserMaster,
+} from 'state/selectors';
+
+class SiteOwnership extends PureComponent {
+	renderPlaceholder() {
+		return (
+			<Card className="manage-connection__card site-settings__card is-placeholder">
+				<div />
+			</Card>
+		);
+	}
+
+	renderConnectionDetails() {
+		const {
+			site,
+			siteIsConnected,
+			siteIsInDevMode,
+			translate,
+			userIsMaster,
+		} = this.props;
+
+		if ( siteIsConnected === false ) {
+			return translate( 'The site is not connected.' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Your site is now connected!' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/site-settings-manage-connection-page)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).